### ISSUE TITLE
Wallet

### ISSUE THREAD

**Issue by D4T4R00T** [D4T4R00T/CryData#5]
/wallet
GET = Lista todos os balanços das wallets

**Comment by D4T4R00T** [comment]
![image](https://user-images.githubusercontent.com/38360096/39401107-23108b48-4b14-11e8-8583-e65b8541207d.png)

**Comment by D4T4R00T** [comment]
**Headers:**
- Content-Type: application/json
- auth_token: <auth_token>

**Comment by D4T4R00T** [comment]
**exemplo do return:**
```JSON
{
  "bitfinex": [
    {
      "time": "2018-04-28T22:05:59.555165",
      "available": 0.0866362,
      "currency": "BCH",
      "balance": 0.0866362,
      "type": "exchange"
    },
    {
      "time": "2018-04-28T22:05:59.555165",
      "available": 64.8343333,
      "currency": "EOS",
      "balance": 64.8343333,
      "type": "exchange"
    },
    {
      "time": "2018-04-28T22:05:59.555165",
      "available": 0.87981391,
      "currency": "ETH",
      "balance": 0.87981391,
      "type": "exchange"
    },
    {
      "time": "2018-04-28T22:05:59.555165",
      "available": 83.50336904,
      "currency": "IOT",
      "balance": 83.50336904,
      "type": "exchange"
    },
    {
      "time": "2018-04-28T22:05:59.555165",
      "available": 153.68205239,
      "currency": "USD",
      "balance": 200.31645239,
      "type": "exchange"
    },
    {
      "time": "2018-04-28T22:05:59.555165",
      "available": 199.8,
      "currency": "XRP",
      "balance": 199.8,
      "type": "exchange"
    }
  ]
}
```
